svc-lighthouse-api/src/test/resources/observation-response-bundle-test.json

```diff
@@ -0,0 +1,235 @@
+                "code": "vital-signs",
+                "display": "Vital Signs"
+              }
+            ],
+            "text": "Vital Signs"
+          }
+        ],
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "85354-9",
+              "display": "Blood pressure systolic and diastolic"
+            }
+          ],
+          "text": "Blood pressure systolic and diastolic"
+        },
+        "subject": {
+          "reference": "https://sandbox-api.va.gov/services/fhir/v0/r4/Patient/9000682",
+          "display": "Mr. Florentino8 Raynor401"
+        },
+        "effectiveDateTime": "2011-03-31T12:24:55Z",
+        "issued": "2011-03-31T12:24:55Z",
+        "result": null,
+        "performer": [
+          {
+            "reference": "https://sandbox-api.va.gov/services/fhir/v0/r4/Organization/I2-TYIC2AW2NXNADER4SKRKJQZWRE000000",
+            "display": "LYONS VA MEDICAL CENTER"
+          },
+          {
+            "reference": "https://sandbox-api.va.gov/services/fhir/v0/r4/Practitioner/I2-QXZOEMHBZNNC7BUGOTHVWYSZAI000000",
+            "display": "DR. JOHN248 SMITH811 MD"
+          }
+        ],
+        "component": [
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "8480-6",
+                  "display": "Systolic blood pressure"
+                }
+              ],
+              "text": "Systolic blood pressure"
+            },
+            "valueQuantity": {
+              "value": 101,
+              "unit": "mm[Hg]",
+              "system": "http://unitsofmeasure.org",
+              "code": "mm[Hg]"
+            },
+            "valueCodeableConcept": null,
+            "valueString": null,
+            "valueInteger": 0,
+            "valueRange": null
+          },
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "8462-4",
+                  "display": "Diastolic blood pressure"
+                }
+              ],
+              "text": "Diastolic blood pressure"
+            },
+            "valueQuantity": {
+              "value": 71,
+              "unit": "mm[Hg]",
+              "system": "http://unitsofmeasure.org",
+              "code": "mm[Hg]"
+            },
+            "valueCodeableConcept": null,
+            "valueString": null,
+            "valueInteger": 0,
+            "valueRange": null
+          }
+        ],
+        "text": null,
+        "valueQuantity": null,
+        "interpretation": null,
+        "referenceRange": null,
+        "valueCodeableConcept": null,
+        "valueString": null,
+        "valueInteger": 0,
+        "valueRange": null
+      },
+      "search": {
+        "mode": "match"
+      }
+    }
+  ]
+}
```

Comment:
Not sure why you always miss the eol at the end files. Is that a setting in your IDE?